src/pepr/policies/storage.ts

```diff
@@ -1,7 +1,8 @@
 import { a } from "pepr";
 
 import { When, containers, volumes } from "./common";
```

Comment:
nit, but for some reason our rules don't pick up import organization rn so if I organize imports in vscode, this import order changes.

Reply:
added this to the `.vscode/settings.json`

```json
"[typescript]": {
    "editor.codeActionsOnSave": {
        "source.organizeImports": "always"
    }
  }
  ```